pom.xml

```diff
@@ -536,6 +536,8 @@
             <configuration>
               <environmentVariables>
                 <HAUTH_VERSION>${project.version}</HAUTH_VERSION>
+                <!-- Default Docker bridge host IP -->
+                <HTTP_HOST>172.17.0.1</HTTP_HOST>
```

Comment:
My guess is that this is the reason for the current build failure. It succeeds locally though, and I am able to reproduce this issue locally by changing the value to e.g. `127.0.0.1` (and in fact, was seeing the issue during development when the env var wasn't set). I guess the Docker bridge is configured differently for GHA VMs.

I guess the test code should get the value programmatically rather than needing it to be passed in, but I couldn't figure out how to do that. Help?

Reply:
(See https://github.com/UCLALibrary/hauth/pull/16/checks?check_run_id=3758817357#step:6:324 for the failure message.)

Reply:
`docker inspect`?

Reply:
Ooh, okay -- thanks!